### ISSUE TITLE
Scripts to mask pretraining.txt

### ISSUE THREAD

**Issue by upcwanghaibo** [CodeReviewAutomation/code_review_automation#1]
In the code/Preprocessing.ipynb, it produces merged pre-training dataset pretraining.txt and trains a SentencePiece tokenizer, but there is no scripts about how to mask pretraining.txt to produce the pretraining.tsv which is used directly in the PreTraining.ipynb. Can you upload the scripts about masking pretraining.txt to produce pretraining.tsv? Thanks.
